container_main.py

```
@@ -66,19 +66,33 @@ def run_scheduler():
+                "gunicorn",
+                "--bind",
+                f"0.0.0.0:{port}",
+                "--workers",
+                "4",
+                "--access-logfile",
+                "-",
+                "--error-logfile",
+                "-",
+                "api_server:app",
+            ]
+            subprocess.run(cmd, check=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)